packages/reactive-element/src/reactive-element.ts

```diff
@@ -666,6 +666,9 @@ export abstract class ReactiveElement
 
   addController(controller: ReactiveController) {
     (this.__controllers ??= []).push(controller);
+    if (this.isConnected) {
```

Comment:
We could almost use `__enableConnection` as a flag that tracks connection state, to avoid the DOM binding, except we don't start that until first disconnection. It feels like there should be a factoring that combines that logic with the initial connection holdoff, and then a flag might come for free out of it...?

Reply:
I don't think we should bother. This change wasn't measurable in benchmarking.

Reply:
Ok, could always change later.